setup.py

```diff
@@ -14,7 +14,7 @@
 
 import setuptools
 
-with open('README.md', 'r') as fh:
+with open('README.md', 'r', encoding="utf-8") as fh:
```

Comment:
I already picked single quotes for the project (not sure it was the right call). But let's stick with it!

Reply:
ok for me 👍